src/pages/gateway/command-reference.md

```diff
@@ -48,6 +48,55 @@ The following example creates the environment in the `mesh_examples` subdirector
 Local workspace created successfully
 ```
 
+## aio api-mesh:run
+
+<InlineAlert variant="info" slots="text"/>
+
+This command is currently in beta.
+
+[Deploys a mesh locally](./developer-tools.md#create-a-local-environment). You only need to run this command if you want to work with your mesh locally for testing. Run `aio api-mesh:init` before running this command.
+
+### Usage
+
+```bash
+aio api-mesh:run [FILE]
+```
+
+### Flags
+
+The following arguments are all optional.
+
+`-p` or `--port` allows you to specify the port number for your local environment. The default is `5000`.
+
+`--debug` enters debug mode. Debugging is disabled by default. To use debugging, add the following configuration to your `launch.json` file:
```

Comment:
```suggestion
`--debug` enters debug mode. Debugging is disabled by default. To use debugging in an IDE such as Visual Studio Code, add the following configuration to your `launch.json` file:
```

Reply:
Yup, `launch.json` is VS Code specific option. We are not endorsing or prioritizing any IDE but better to give an example config of how they can do in VS Code which most of our users and partners seem to be using.

Also worth removing `Debugging is disabled by default.`. It is not disabled. Using the above config will support native debugging on VS Code instead of using a custom/3rd party debugger.